css/layout.css

```diff
@@ -16,35 +16,31 @@
 }
 
 .column-fourth {
-  flex-basis: 25%;
+  flex-basis: 100%;
 }
 
 .column-half {
-  flex-basis: 50%;
+  flex-basis: 100%;
```

Comment:
You can target these utility classes together (separated by a comma) if they are doing the same thing rather than having separate CSS rulesets for it.

```
.column-fourth,
.column-half,
.column-full {
  flex-basis: 100%;
  }
  ```

Reply:
Updated this.  Thanks.